data/google-analytics.json

```diff
@@ -15,7 +15,7 @@
       "key": "gtag"
     },
     {
-      "code": "window[{{l}}]=window[{{l}}]||[];window['gtag-'+{{l}}]=function (){window[{{l}}].push(arguments);};window['gtag-'+{{l}}]('js',new Date());window['gtag-'+{{l}}]('config',{{id}})",
+      "code": "window[{{l}}]=window[{{l}}]||[];window['gtag-'+{{l}}]=function (){window[{{l}}].push(arguments);};window['gtag-'+{{l}}]('consent', 'default', {'ad_user_data': 'denied','ad_personalization': 'denied','ad_storage': 'denied','analytics_storage': 'denied','wait_for_update': 500,});window['gtag-'+{{l}}]('js',new Date());window['gtag-'+{{l}}]('config',{{id}})",
```

Comment:
My only concern with this is if anyone wants to have different consent options in the future. Instead of hardcoding, what do you think about including `consent` as optionalParams instead?

Something like:

```
      //...
      "optionalParams": {
        "l": "dataLayer",
        "consentType": "default",
        "consentValues": {},
      },
 ```
 
 CC @flashdesignory @felixarntz

Reply:
definitively :+1: